### PR TITLE
fix[test]: fix a boundary case in decimal fuzzing

### PR DESCRIPTION
**Pull request by charles-cooper**
change to not test the boundary condition at all, but rely on whether `ExprVisitor().visit()` throws an exception or not.

### Commit message

Commit message for the final, squashed PR. (Optional, but reviewers will appreciate it! Please see [our commit message style guide](../../master/docs/style-guide.rst#best-practices-1) for what we would ideally like to see in a commit message.)

### Cute Animal Picture

Put a link to a cute animal picture inside the parenthesis-->

### CODE REVIEW

File: tests/unit/ast/nodes/test_fold_binop_decimal.py [hunk 1]
```diff
@@ -81,11 +84,13 @@ def foo({input_value}) -> decimal:
...
         vyper_ast = parse_and_fold(literal_op)
         expr = vyper_ast.body[0].value
+
+        # check invalid intermediate values
         ExprVisitor().visit(expr, DecimalT())
+
         new_node = expr.get_folded_value()
         expected = new_node.value
-        lo, hi = DecimalT().decimal_bounds
-        is_valid = lo <= expected < hi
```
Comment (fubuloubu) [inline tests/unit/ast/nodes/test_fold_binop_decimal.py:88]: This no longer needs to be checked?

**Review by fubuloubu — APPROVED** [review]

File: tests/unit/ast/nodes/test_fold_binop_decimal.py (hunk 1 above)
Comment (charles-cooper) [reply tests/unit/ast/nodes/test_fold_binop_decimal.py:88]: it's checked by `ExprVisitor().visit()`, which will throw an exception if oob
